### PR TITLE
port some meta examples

### PR DESCRIPTION
**Pull request by LiuYuHui**

### CODE REVIEW

**Comment by gf712** [comment]
Could you also add the data files please?

File: examples/meta/src/kernel/poly.sg.in [hunk 1]
```diff
@@ -0,0 +1,18 @@
+File traindat = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
```
Comment (karlnapf) [inline examples/meta/src/kernel/poly.sg.in]: polynomial

File: examples/meta/src/kernel/inverse_multiquadric.sg.in [hunk 2]
```diff
@@ -0,0 +1,19 @@
+File traindat = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
```
Comment (karlnapf) [inline examples/meta/src/kernel/inverse_multiquadric.sg.in]: multi_quadratic

**Review by karlnapf — COMMENTED** [review]
Mini comments otherwise fine from my side

File: examples/meta/src/kernel/inverse_multiquadric.sg.in (hunk 2 above)
Comment (LiuYuHui) [reply examples/meta/src/kernel/inverse_multiquadric.sg.in]: Do you mean change file name to multi_quadratic?

Comment (karlnapf) [reply examples/meta/src/kernel/inverse_multiquadric.sg.in]: Yes both example and data

File: examples/meta/src/kernel/multi_quadratic.sg.in [hunk 3]
```diff
@@ -0,0 +1,19 @@
+File traindat = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
```
Comment (karlnapf) [inline examples/meta/src/kernel/multi_quadratic.sg.in]: quick question: why remove the "inverse" in the filename?

File: examples/meta/src/kernel/inverse_multiquadric.sg.in (hunk 2 above)
Comment (karlnapf) [reply examples/meta/src/kernel/inverse_multiquadric.sg.in]: sorry, I see now. I of coure mean "inverse_multi_quadratic"
I should have been more clear, sorry about that hassle :)

**Comment by karlnapf** [comment]
Thanks!

**Pull request merged**